palette/transaction-payment/rpc/runtime-api/src/lib.rs

```diff
@@ -27,7 +27,6 @@ use serde::{Serialize, Deserialize};
 /// Some information related to a dispatchable that can be queried from the runtime.
 #[derive(Eq, PartialEq, Encode, Decode, Default)]
 #[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
-#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
```

Comment:
Not sure this specific line should be reverted - this doesn't affect anything and as it stands the naming here is inconsistent with what is used everywhere else.